TTTSwiftTests/TTTAcceptanceTests.swift

```diff
@@ -5,9 +5,9 @@ import TTTSwift
 class TTTAcceptanceTests: QuickSpec {
     override func spec() {
         
-        describe("a full game of tic-tac-toe between two first available space computers plays through and exits properly") {
+        describe("acceptance tests") {
             
-            it("plays the game") {
+            it("a full game of tic-tac-toe between two first available space computers plays through and exits properly") {
```

Comment:
Keep in mind that you test descriptions should read as valid english. `it a full game...` "A full game" seems to be the thing you are describing.